src/k8s/pkg/k8sd/database/sql/queries/cluster-configs/insert-worker-token.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO
+    cluster_configs(key, value)
+VALUES
+    ("worker-token", ?)
```

Comment:
So, on the worker-node we store the token of for that node in `k8sd` right? 
Why are we doing this? For the control-planes it makes sense to store it in the DB so that other nodes can access it, but wouldn't a file be sufficient for the worker?
Also, it feels a bit weird to use the `cluster-configs` DB as a version->yaml store in the control-plane and as a key->value store in the worker node.

Reply:
No, this is stored on the control plane nodes to authenticate requests to `POST /k8sd/worker/info`.

Ι re-used `cluster_configs` since it is already a "generic" key-value place for configs. Kept it out of the `ClusterConfig` struct, in order to have runtime secrets and persistent cluster configs separate. Note that we have separate methods to access this, so it's fine if we are using the same table, our "API" ensures we never break things.

Reply:
currently only one worker token is used, indeed. the token is created here https://github.com/canonical/k8s-snap/blob/b9dcfc2176b882bfebb1bcec53902880cacd4780/src/k8s/pkg/k8sd/database/worker.go#L39-L65, therefore the race condition would occur if:

- two control plane nodes attempt to `add-node --worker` at the same time.
- both database transactions start before the other finishes, and see an empty token
- both set the token to something, then one of the writes is lost.

This is already a quite hard situation to get into, and the single-writer of sqlite should be enough protection for _most_ cases.

Multiple workers can still join, `CheckWorkerToken` will work without issues.

Reply:
makes sense - thanks!